master/getting-started/kubernetes/index.md

```diff
@@ -1,31 +1,68 @@
 ---
 title: Kubernetes
 ---
-Calico can be used as a network plugin for Kubernetes to provide connectivity and network policy in a Kubernetes cluster.
-Calico allows you to enforce fine-grained network security policy natively in Kubernetes. The video below shows a quick demonstration of Calico policy in action.
 
-[![IMAGE ALT TEXT](http://img.youtube.com/vi/OE1n5PWtvMM/0.jpg)](http://www.youtube.com/watch?v=OE1n5PWtvMM "Calico network policy on Kubernetes")
+Calico drives pod connectivity and implements network policy for Kubernetes clusters.
+Calico allows you to enforce fine-grained network security policy natively in Kubernetes.
 
-## Getting Started
+Use the navigation bar on the left to view information on Calico for Kubernetes,
+or continue reading for recommended guides to get started.
 
-The following guides help you get started with Calico.
+## Installation Guides
 
-Integration guide
-------------------
+There are several ways to install Calico in Kubernetes.
 
-- [Integration Guide]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/): Discusses adding Calico to an existing cluster, or integrating Calico with your own install scripts.
+#### [Hosted Install Guide (Beta)]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/hosted)
```

Comment:
Could you remove the "beta" tag from this?  Worth including the v1.4+ note still, but we are moving this out of beta for v2.0